app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
```

Comment:
This is strange in that if the rule does NOT have a mapping you end up with something like this:
```html
<h5 title="Mapping">
    Mapping:
</h5>
```